code/datums/spells/construct_spells.dm

```diff
@@ -64,7 +64,11 @@
 	action_icon_state = "summonsoulstone"
 	action_background_icon_state = "bg_demon"
 
+	if(iscultist(user) || iswizard(user))

```

Comment:
Whelp. Uh... ok maybe I can make a second summon spell that makes normal stones, and give it to non-cult artis when they are made.